worker/src/RTC/PipeTransport.cpp

```diff
@@ -168,7 +168,12 @@ namespace RTC
 			}
 
 			tuple = FBS::Transport::CreateTupleDirect(
-			  builder, localIp.c_str(), this->udpSocket->GetLocalPort(), "", 0, FBS::Transport::Protocol::UDP);
+			  builder,
+			  localIp.c_str(),
+			  this->udpSocket->GetLocalPort(),
+			  nullptr,
```

Comment:
This is a fix here, remote ip is optional and hence must be `nullptr` rather than empty string here.